File: CubeSummationApp.WebUI/ClientApp/src/app/models/Query.ts
import { IQuery } from './IQuery';
import { ICoordinate } from './ICoordinate';
import { Coordinate } from './Coordinate';

export class Query implements IQuery {
  Tipo: string;
  CoordenadaInicial: ICoordinate;
  CoordenadaFinal: ICoordinate;

  constructor() {
    this.CoordenadaInicial = new Coordinate();
    this.CoordenadaFinal = new Coordinate();
  }
}

File: CubeSummationApp.Application.Entities/Entities/Query.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CubeSummationApp.Application.Entities
{
    public class Query
    {
        public string Tipo { get; set; }
        public Coordinate CoordenadaInicial { get; set; }
        public Coordinate CoordenadaFinal { get; set; }
    }
}
